pointerevents/pointerevent_sequence_at_implicit_release-manual.html

```diff
@@ -0,0 +1,112 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Pointer Event: Event sequence at implicit release</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
+    <link rel="author" title="Google" href="http://www.google.com "/>
+    <meta name="assert" content="When a captured pointer is implicitly released, the boundary events should follow the lostpointercapture event."/>
+    <link rel="stylesheet" type="text/css" href="pointerevent_styles.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script type="text/javascript" src="pointerevent_support.js"></script>
+    <script type="text/javascript">
+      var test_pointerEvent_0 = async_test("Event sequence at implicit release after click");
+      var test_pointerEvent_1 = async_test("Event sequence at implicit release after drag");
+
+      add_completion_callback(end_of_test);
+
+      var detected_pointertypes = {};
+
+      var test_num = 0;
+      var event_log = [];
+      var start_logging = false;
+      var seen_pointercancel = false;
+
+      function end_of_test() {
+        showLoggedEvents();
+        showPointerTypes();
+      }
+
+      function end_of_current_test() {
+        if (test_num == 0) {
+          var expected_events = "pointerup, lostpointercapture, pointerout, pointerleave";
```

Comment:
This doesn't seem right for the mouse case. If we just click in the black why do we get pointerout/leave for mouse?